### PR TITLE
Enable testcafe travis

### PR DESCRIPTION
**Pull request by haishengwu-okta**
cc @mauriciocastillosilva-okta @magizh-okta

### CODE REVIEW

File: webpack.playground.config.js [hunk 1]
```diff
@@ -11,14 +11,16 @@ const WIDGET_RC = '.widgetrc';
...
 let widgetRc = {
   widgetOptions: {
-    baseUrl: 'https://{yourOktaDomain}',
+    baseUrl: 'http://localhost:3000',
     logo: '/img/logo_widgico.png',
     logoText: 'Windico',
     features: {
       router: true,
       rememberMe: true,
+      securityImage: true,
```
Comment (mauriciocastillosilva-okta) [inline webpack.playground.config.js]: security image has not worked for me before locally... not sure if that is fixed or maybe it was just me, but if it doesn't work easily for other people, should we then leave it disabled?

Comment (nikhilvenkatraman-okta) [reply webpack.playground.config.js]: why do we need this to be true ?  securityImage does not work for new pipeline anyways

File: webpack.playground.config.js [hunk 2]
```diff
@@ -11,14 +11,16 @@ const WIDGET_RC = '.widgetrc';
 
 let widgetRc = {
   widgetOptions: {
-    baseUrl: 'https://{yourOktaDomain}',
+    baseUrl: 'http://localhost:3000',
```
Comment (magizh-okta) [inline webpack.playground.config.js]: Right now we are having tests for idx pipeline powered by mock server right ? And if no widgetrc is provided by default playground opens up with idx flow pointing to mock server right.

Comment (haishengwu-okta) [reply webpack.playground.config.js]: correct.

File: webpack.playground.config.js (hunk 1 above)
Comment (haishengwu-okta) [reply webpack.playground.config.js]: - @mauriciocastillosilva-okta it does given we mocked `http://localhost:3000/login/getimage` request
- @nikhilvenkatraman-okta the `widgetrc` is not dedicated for OIE though. but I'll remove it from the default generator now.

File: webpack.playground.config.js [hunk 3]
```diff
@@ -11,14 +11,15 @@ const WIDGET_RC = '.widgetrc';
...
     logo: '/img/logo_widgico.png',
     logoText: 'Windico',
     features: {
       router: true,
```
Comment (magizh-okta) [inline webpack.playground.config.js]: Does these features make sense in idx flows ?

Comment (haishengwu-okta) [reply webpack.playground.config.js]: probably not. the `widgetrc` is intend for both v1/v2 and we didn't clearly call out the difference.

**Pull request merged**